### DISCUSSION TITLE
porosity screen - free area %

### DISCUSSION THREAD

**Discussion by JiangH** [firemodels/fds#13317] [Q&A]
I am using porosity screen function, as a way to mimic a false ceiling with various percentage free area. However when i adopt a low % free area, my wall time increased dramatically whenever i go below 30% free area. 
Is this a limitation of the modelling equations? or there is a better way to do what i am doing ? 
I  want to simulate a porous screen between two compartments for a range of % free area, 10-100%

**Comment by drjfloyd** [comment]
Make sure you are using the current release of FDS. 

The lower the free area, the more the drag and the higher the likelihood that a particle based CFL will start to limit the timestep.

As stated in the guide, the screen drag routine is based on data for window screens with 30 to 60 % porosity. If you are dropping below 30 % free area or going above 60 %, you are extrapolating the correlation beyond the range of data used to develop it. You may not be getting reasonable drag beyond the limits of the correlation.

Window screens are not perforated ceiling panels. I would recommend you test your inputs in case similar to the screen drag verification cases to verify that you are getting an appropriate pressure drop for the various free area fractions for your ceiling panels. The thickness of your ceiling panel is likely much larger than the typical wire thickness for a window screen which could also be a factor. 

You may need to develop custom drag parameters to reflect the manufacturer reported pressure drop through the ceiling.

**Reply by JiangH** [reply]
thanks, i'll test the drag and pressure across the compartments. i'll keep posted here on progress.

**Comment by mcgratta** [comment]
Here's tip -- put a `slice file (`SLCF` ) with `VECTOR=T` perpendicular to the screen. If you begin to see the vectors acting erratically, you have probably set the open area too low. FDS will not converge nicely to zero open area.

**Reply by JiangH** [reply]
thanks. i have this slice in and do not spot any erratic behaviour of the vectors. but like the post above, it may be down the a convergence issue and the solver is struggling to converge to a value, hence the increasing wall time. i'll do some more testing and keep posted.

**Comment by Er9y714** [comment]
Can you also try running the case with an input of PARTICLE_CFL_MAX = 0.9 in the &MISC line?
